tests/commentsCounter.test.js

```diff
@@ -0,0 +1,15 @@
+import countComments from '../src/modules/commentsCounter.js';
+
+describe('App can render tasks', () => {
+  test('Should count homepage items', () => {
+    document.body.innerHTML = '<ul class="comments">'
+      + '  <li class="comment"><li>'
+      + '  <li class="comment"><li>'
+      + '  <li class="comment"><li>'
+      + '  <li class="comment"><li>'
+      + '</ul>';
+
+    const commentsCount = countComments();
+    expect(commentsCount).toBe(4);
+  });
```

Comment:
- [ ] Great job adding tests for the counters, but let's think of a scenario where there are no comments for a movie. In this case, it could be great to start with a test that will return 0 because there is no comment(s), then add comment(s) Examples that should return the counter e.g. 4.